### ISSUE TITLE
Connect paid plans on the website to the sign in user flow

### ISSUE THREAD

**Issue by geoffreyvancassel** [labelflow/labelflow#816]
- [ ] Replace on the website the "Request access" call to action on Starter and Pro plan by "Start Free Trial" 
- [ ] Write below the Starter and Pro plan that the trial is 14 days
- [ ] User flow
  - [ ] Display sign-in modal
  - [ ] Show workspace creation modal and create workspace with the corresponding plan
- [ ] Adjust the plan type in settings from community to starter/pro when these plans are clicked

When people will have exceeded the 1000 images we will ask them to pay manually in the beginning to keep things simple

Depends on #799

**Comment by nicOwlas** [comment]
Duplicate of #531
